### PR TITLE
Replace division with floor division as of PEP-238

### PR DESCRIPTION
**Pull request by Arusekk**
This will also get us closer to python3.
See https://www.python.org/dev/peps/pep-0238/

### CODE REVIEW

File: pwnlib/constants/linux/s390.py [hunk 1]
```diff
@@ -511,7 +511,7 @@
 _SIGCONTEXT_NSIG = Constant('_SIGCONTEXT_NSIG',64)
 _SIGCONTEXT_NSIG_BPW = Constant('_SIGCONTEXT_NSIG_BPW',64)
 __SIGNAL_FRAMESIZE = Constant('__SIGNAL_FRAMESIZE',160)
-_SIGCONTEXT_NSIG_WORDS = Constant('_SIGCONTEXT_NSIG_WORDS',(64 / 64))
+_SIGCONTEXT_NSIG_WORDS = Constant('_SIGCONTEXT_NSIG_WORDS',(64 // 64))
```
Comment (zachriggle) [inline pwnlib/constants/linux/s390.py]: This is auto-generated from C source.  Please revert all changes to pwnlib/constants.

**Review by zachriggle — CHANGES_REQUESTED** [review]
Please add `from __future__ import division` to all source files, and ensure that the tests pass.

Thanks!

**Comment by Arusekk** [comment]
Do you mean all python files by source files or just the ones involved in this commit?

**Comment by zachriggle** [comment]
All source files that are not auto-generated (e.g. not pwnlib/data or pwnlib/constants).  There should be a flag you can pass to `six` or `futurize` that does this automagically.

*Edit*: Specifically, all of the files *should* already have `from __future__ import absolute_import` -- just add it near that statement.

**Comment by Arusekk** [comment]
The tests successfully pass (but for the `runner` module, which is having some problems with coreutils binaries, possibly a wrong dynamic linker).

I'm ready with further adaptations for python3, in branch Arusekk/pwntools:full-3-compat
Just don't want to dump everything at once, nor to spam the pulls board.

**Comment by zachriggle** [comment]
Thanks!

**Pull request merged**